src/CppUTest/Utest.cpp

```diff
@@ -676,6 +696,16 @@ TestTerminatorWithoutExceptions::~TestTerminatorWithoutExceptions()
 {
 }
 
+void CrashingTestTerminator::exitCurrentTest() const
+{
+    UtestShell::crash();
+    NormalTestTerminator::exitCurrentTest();
```

Comment:
Why do we still want to do the exitCurrentTest?

Reply:
This is needed in part due to the way that we test the crashing functionality. We do the same thing in the Mock TestTerminator class (albeit the crash is conditional based on the value of a Boolean in that case.)
Also, since the crash function can be changed, better to fail the test if the crash function returns than go back to the test and keep running it...

Reply:
Ok, lets keep that then for now.